JOSS/paper.md

```diff
@@ -82,10 +82,12 @@ Two procedures are followed when assigning the various urban canopy parameters t
 
 **Procedure 1**: **Morphological parameters** are assigned directly to the high-resolution LCZ map, and are afterwards aggregated to the lower-resolution WRF grid. As a result, the method produces a unique urban morphology parameter value for each WRF grid cell. This was found to be more efficient in reproducing urban boundary layer features, especially in the outskirts of the city [@Zonato2020], and is in line with the [WUDAPT-to-COSMO](https://github.com/matthiasdemuzere/WUDAPT-to-COSMO) routine [@Varentsov2020].
 
-Morphological urban canopy parameter values are provided in `LCZ_UCP_default.csv`, and are generally based on values provided in @Stewart2012 and @Stewart2014. In addition:
+Morphological urban canopy parameter values are provided in `LCZ_UCP_default.csv`, and are generally based on values provided in @Stewart2012 and @Stewart2014. Note however that the values of MH_URB2D_MIN, MH_URB2D, MH_URB2D_MAX for LCZ 7 are set to 4, 5 and 6 m instead of 2, 3 and 4 m, because the minimum building height that can assigned to BEP-BEM is 5m if dz_u = 5m (standard value) is used.
```

Comment:
typo here - missing **be**? the minimum building height that can **be** assigned

Reply:
yes, good catch! Please adjust!